src/Utilities/UnitTests/ToolTask_Tests.cs

```diff
@@ -27,7 +27,11 @@ public MyTool()
                 : base()
             {
                 _fullToolName = Path.Combine(
+#if FEATURE_SPECIAL_FOLDERS
                     Environment.GetFolderPath(Environment.SpecialFolder.System),
```

Comment:
@AndyGerlicher The reason it's currently like it is is that the SpecialFolder enum is nested inside the Environment class, so I couldn't write an abstraction that just passes it through.  Casting from one enum to another would probably be less messy overall than having #if's sprinkled everywhere we call this code.